R/mapDGEobj.R

```diff
@@ -50,12 +56,16 @@
 #' @import magrittr
 #' @importFrom assertthat assert_that
 #' @importFrom igraph graph_from_data_frame
+#' @importFrom canvasXpress canvasXpress
+#' @importFrom htmlwidgets JS
 #'
 #' @export
-mapDGEobj <- function(dgeObj, directed = TRUE) {
+mapDGEobj <- function(dgeObj, plotType = "canvasXpress", directed = TRUE) {
 
     assertthat::assert_that("DGEobj" %in% class(dgeObj),
                             msg = "dgeObj must be of class 'DGEobj'.")
+    assertthat::assert_that(plotType %in% c("ggplot", "canvasXpress"),
+                            msg = "Plot type must be either ggplot or canvasXpress.")
```

Comment:
since CX is default, list that first.  If this message needs to be fixed elsewhere please add that task.

Reply:
Done